src/_pylibmcmodule.c

```diff
@@ -2491,9 +2554,14 @@ static void _make_excs(PyObject *module) {
     PylibMCExc_Error = PyErr_NewException(
             "pylibmc.Error", NULL, NULL);
 
+    PylibMCExc_CacheMiss = PyErr_NewException(
```

Comment:
I mean that it seems strange to not inherit from the common base exception in pylibmc.

> On 11 okt. 2015, at 14:55, Shivaram Lingamneni <notifications@github.com> wrote:
> 
> In src/_pylibmcmodule.c:
> 
> > @@ -2491,9 +2554,14 @@ static void _make_excs(PyObject *module) {
> >      PylibMCExc_Error = PyErr_NewException(
> >              "pylibmc.Error", NULL, NULL);
> >  
> > +    PylibMCExc_CacheMiss = PyErr_NewException(
> 
> Should it inherit from BaseException instead of Exception? This seems not recommended.
> 
> For comparison, StopIteration inherits from Exception (but not from StandardError).
> 
> —
> Reply to this email directly or view it on GitHub.
>